src/__tests__/domToHtml.test.tsx

```diff
@@ -1,18 +1,76 @@
 import { test, expect } from '@playwright/experimental-ct-react';
 
-import { TestComponent } from './utils';
+import { TestComponent, CanvasTest, FullTest, SvgTest } from './utils';
 
 test.describe('test domToHtml', () => {
-  test('simple test', async ({ mount }) => {
+  test('basic', async ({ mount }) => {
     const component = await mount(
       <TestComponent>
         <div>
-          <span>test</span>
+          <span>text</span>
         </div>
       </TestComponent>,
     );
+    const test = component.locator('#test');
+    await test.screenshot({ path: './src/__tests__/snapshots/basic.png' });
```

Comment:
I'm not familiar with screenshot testing. What is the purpose of this line? Looking at playwright's documentation it seems you can just call `toHaveScreenshot` and it will create the image if it does not exist yet.

Reply:
Test page contain three sections
test => the original dom
html => text area to get html code generated with domToHtml
result=> the created dom with the html generated with domToHtml

so here i saved screenshot of test and compare it to result screenshot